Cloudinary/Classes/Core/Features/UploadWidget/WidgetElements/CLDWidgetPreviewCollectionCell.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  CLDWidgetPreviewCollectionCell.swift
+//
+//  Copyright (c) 2020 Cloudinary (http://cloudinary.com)
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in all
+//  copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+//  SOFTWARE.
+//
+
+import UIKit
+
+class CLDWidgetPreviewCollectionCell: UICollectionViewCell {
+    
+    var imageView: UIImageView
+    
+    // MARK: - init
+    override init(frame: CGRect) {
+        
+        self.imageView = UIImageView()
+        
+        super.init(frame: frame)
+        
+        initialize()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - private methods
+    private func initialize() {
+        
+        addSubview(imageView)
+        addImageViewProperties()
+        addImageViewConstraints()
+    }
+    
+    private func addImageViewProperties() {
+        imageView.contentMode = .scaleAspectFit
+    }
+    
+    private func addImageViewConstraints() {
+        
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        
+        let constraints = [
+            NSLayoutConstraint(item: imageView, attribute: .leading, relatedBy: .equal, toItem: contentView, attribute: .leading, multiplier: 1, constant: 0),
```

Comment:
Do we have a good reason to define the View and it's constraints in code like that? Why not use the visual designer? The same questions for `CLDWidgetPreviewViewController` as well.

Reply:
i'll see if we can simplify the expression, maybe by using the visual design language